evaluation/benchmarks/run.seq.sh

```diff
@@ -52,7 +52,11 @@ oneliners(){
     IFS=";" read -r -a script_input_parsed <<< "${script_input}"
     script="${script_input_parsed[0]}"
     input="${script_input_parsed[1]}"
-    export IN="$PASH_TOP/evaluation/benchmarks/oneliners/input/$input"
+    if [[ "$1" == "--small" ]]; then
```

Comment:
It is a little ugly that this needs to be duplicated in both run.seq.sh and run.par.sh. Could you think of a way to abstract this behavior (and many others that are the same) in a common function/file that both run.par.sh and run.seq.sh use?

Reply:
That's quite a good point. I could create a new function that will be called from each benchmark during its startup. Moreover, I am thinking of moving the `export`- ing of the respective variables (`IN IN_PRE` etc to the `setup.sh`). What do you think?

Reply:
My only issue with moving exporting to setup is that one might want to setup different benchmark suites all together in the beginning and then run them one by one. This would break the flow. I think that exporting of respective variables should happen as close as possible to the actual execution (maybe it can be a function that is defined in setup.sh and then sourced, but exporting should not be tied with downloading inputs IMO.